packages/grid-pro/src/lit/column-renderer-directives.d.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { TemplateResult } from 'lit';
+import { DirectiveResult } from 'lit/directive';
+import { GridItemModel } from '@vaadin/grid';
+import { LitRendererDirective } from '@vaadin/lit-renderer';
+import { GridProEditColumn } from '../vaadin-grid-pro-edit-column.js';
+
+export type GridProColumnEditModeLitRenderer<TItem> = (
+  item: TItem,
+  model: GridItemModel<TItem>,
+  column: GridProEditColumn,
+) => TemplateResult;
+
+export declare class GridProColumnEditModeRendererDirective<TItem> extends LitRendererDirective<
```

Comment:
**note:** It was not possible to extend `AbstractGridColumnBodyRendererDirective` because it only allows `rendererProperty` to be `'renderer' | 'headerRenderer' | 'footerRenderer'`.